charts/spire/charts/spire-server/templates/post-upgrade-hook.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if eq (.Values.controllerManager.enabled | toString) "true" }}
```

Comment:
Each of the other hook files has the same issues.

Reply:
I really wouldn't make this configurable, this is just a necessary fix until the the controller-manager-webhook fixes it.

Reply:
If we're willing to compromise on a workaround to keep it simple because its temporary, we can also just always set the fail policy to ignore until the fix is out and that's much much simpler then all of these hooks.

Reply:
IMHO this is the real fix. At least after the upgrade there who had the behavior I want.

Reply:
Patching files with hooks that helm manages while the system is running is a hack. We're just debating which hack is best at this point. I'm glad the behavior works for you. I don't think the behavior works for me. So I just want the option to opt out.

Reply:
This is covered with an additional configurable.

Reply:
Not sure I follow. Do you mean you think it can be done as is with one of the configurables, or that you will add the additional configurable?

Reply:
Sorry, typo. *without*. If you set fail policy to ignore the hooks wont run